src/ui-kit/form-controls/autocomplete-multiselect/autocomplete-multiselect.component.ts

```diff
@@ -232,6 +232,9 @@ export class SamAutocompleteMultiselectComponent
     if (this.list.length > 0) {
       this.list = this.sortByCategory(this.list);
     }
+  }
+
+  public ngAfterViewInit(){
```

Comment:
Why are you not adding the implements the interface?

Reply:
updated